pages/products/index.tsx

```diff
@@ -1,23 +1,43 @@
-import { Button, Dropdown, Panel, Small, StatefulTable, Link as StyledLink } from '@bigcommerce/big-design';
+import { Button, Dropdown, Panel, Small, Link as StyledLink, Table, TableSortDirection } from '@bigcommerce/big-design';
 import { MoreHorizIcon } from '@bigcommerce/big-design-icons';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
-import { ReactElement } from 'react';
+import { ReactElement, useState } from 'react';
 import ErrorMessage from '../../components/error';
 import Loading from '../../components/loading';
 import { useProductList } from '../../lib/hooks';
 import { TableItem } from '../../types';
 
 const Products = () => {
+    const [itemsPerPage, setItemsPerPage] = useState(10);
+    const [currentPage, setCurrentPage] = useState(1);
+    const [columnHash, setColumnHash] = useState('id');
+    const [sortDirection, setSortDirection] = useState<TableSortDirection>('ASC');
     const router = useRouter();
-    const { error, isLoading, list = [] } = useProductList();
+    const { error, isLoading, list = [], meta = {} } = useProductList({
+      page: String(currentPage),
+      limit: String(itemsPerPage),
+      sort: columnHash,
+      direction: sortDirection === 'ASC' ? 'asc' : 'desc',  
```

Comment:
I'm converting to lowercase this way because `Table`'s direction attribute needs to be of type `TableSortDirection` which prevents me from using `toLowercase()` without first changing it to type string. This felt like a cleaner option.

Reply:
I believe we can still use `.toLowerCase()` here. We probably don't want to do a manual conversion of strings unless we absolutely have to.
`direction: direction.toLowerCase()`